app/assets/stylesheets/modules/top_navbar.scss

```diff
@@ -38,8 +37,36 @@
  }
 }
 
+.navbar {
+  max-height: 120px;
+
+  .header-bar {
+    flex-wrap: wrap !important;
+  }
+
+  #topnav-container {
+    padding: 2px 0;
+    height: 29px;
+  }
+}
+
 @include media-breakpoint-up(xs) {
   #topnav {
     width: 100%;
   }
 }
+
+@media (min-width: 768px) {
+  .navbar {
+    max-height: 70px;
+  }
+
+  .navbar-nav {
```

Comment:
Should these be scoped to `#topnav` or live in `header_navbar.scss` and scoped there? Or are they more globally applied?

Reply:
Also does not appear to be needed at all right now, so I am just removing it.